### PR TITLE
add back integration site-region updates

### PR DESCRIPTION
**Pull request by StefonSimmons**
### What does this PR do?

fix toc on `/integrations/` pages

### Preview

The H3 and H2 headers equivalent TOC items  are now showing in the TOC.
- https://docs-staging.datadoghq.com/stefon.simmons/fix-site-region-integration-pgs/integrations/azure
- https://docs-staging.datadoghq.com/stefon.simmons/fix-site-region-integration-pgs/integrations/adobe_experience_manager
- https://docs-staging.datadoghq.com/stefon.simmons/fix-site-region-integration-pgs/integrations/apigee
- https://docs-staging.datadoghq.com/stefon.simmons/fix-site-region-integration-pgs/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose
- https://docs-staging.datadoghq.com/stefon.simmons/fix-site-region-integration-pgs/integrations/rsyslog
- https://docs-staging.datadoghq.com/stefon.simmons/fix-site-region-integration-pgs/integrations/syslog_ng
- https://docs-staging.datadoghq.com/stefon.simmons/fix-site-region-integration-pgs/integrations/vmware_tanzu_application_service

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

**Review by alai97 — APPROVED** [review]

**Comment by davidejones** [comment]
According to the last change in #17329 it seems its possible that enabling this again is going to cause link formatter errors. Can we test that @StefonSimmons ?

File: local/bin/py/build/configurations/pull_config_preview.yaml (hunk 1 above)
Comment (StefonSimmons) [reply local/bin/py/build/configurations/pull_config_preview.yaml:3]: @alai97 setting this to true on preview reverts to a cached version that is not desired. i'll try make another PR that sets the cache to true after it is merged to master if that is ok

**Comment by StefonSimmons** [comment]
> r errors. Can we test that @StefonSimmons ?

I believe I got rid of the pre commit hook that changed the links around

> According to the last change in #17329 it seems its possible that enabling this again is going to cause link formatter errors. Can we test that @StefonSimmons ?

hey @davidejones I believe this issue is related to the precommit hook (link checker?) which shouldnt be running since I removed it, right?

**Pull request closed without merging**